easybuild/toolchains/foss.py

```diff
@@ -56,7 +57,7 @@ def __init__(self, *args, **kwargs):
         constants = ('BLAS_MODULE_NAME', 'BLAS_LIB', 'BLAS_LIB_MT', 'BLAS_FAMILY',
                      'LAPACK_MODULE_NAME', 'LAPACK_IS_BLAS', 'LAPACK_FAMILY')
 
-        if self.looseversion > LooseVersion('2021.0'):
+        if self.looseversion > LooseVersion('2021.0') and get_cpu_architecture() != POWER:
```

Comment:
@branfosj We should explain a bit better why we opt out on POWER, please add a comment incl. a reference to https://github.com/easybuilders/easybuild-easyconfigs/issues/12968 and https://github.com/mpimd-csc/flexiblas/issues/17 ?

Reply:
comment added in 8ab4aef